WcaOnRails/app/views/registrations/_payment_form.html.erb

```diff
@@ -1,128 +1,125 @@
+  <%= f.button :button, t('registrations.payment_button_text'), id: :"stripe-button", class: 'btn btn-primary' %>
+  <div id="stripe-elements" style="display: none;">
+    <div class="form-group" id="group-payment-element">
+      <label class="col-sm-2 control-label" for="payment-element"><%= t("registrations.payment_form.labels.payment_information") %></label>
+      <div class="col-sm-9">
+        <div id="payment-element"></div>
+        <p class="help-block"><%= t("registrations.payment_form.hints.payment_information") %></p>
+      </div>
     </div>
-  </div>
-  <div class="form-group" id="group-card-element">
-    <label class="col-sm-2 control-label" for="card-element"><%= t("registrations.payment_form.labels.card_information") %></label>
-    <div class="col-sm-9">
-      <div class="form-control" id="card-element"></div>
-      <p class="help-block"><%= t("registrations.payment_form.hints.card_information") %></p>
+    <div class="form-group">
+      <div class="col-sm-offset-2 col-sm-10 text-danger" id="payment-error">
+      </div>
     </div>
-  </div>
-  <div class="form-group">
-    <div class="col-sm-offset-2 col-sm-10 text-danger" id="card-error">
-    </div>
-  </div>
-  <div class="form-group">
-    <div class="col-sm-offset-2 col-sm-10">
-      <%= f.button :button, t('registrations.payment_button_text'), id: :"card-button", class: 'btn btn-primary' %>
+    <div class="form-group">
+      <div class="col-sm-offset-2 col-sm-10">
+        <%= f.button :button, t('registrations.payment_confirmation_text'), id: :"payment-button", class: 'btn btn-primary' %>
+      </div>
     </div>
   </div>
   <script src="https://js.stripe.com/v3/"></script>
   <script>
-    // From https://stripe.com/docs/stripe-js/reference#checkout-options-locale
-    var supported_locales = ['da', 'de', 'en', 'es', 'fi', 'fr', 'it', 'ja', 'nb', 'nl', 'pl', 'pt', 'sv', 'zh'];
+    // From https://stripe.com/docs/js/appendix/supported_locales
+    var supported_locales = ['ar', 'bg', 'cs', 'da', 'de', 'el', 'en', 'en-GB', 'es', 'es-419', 'et', 'fi', 'fil', 'fr', 'fr-CA', 'he', 'hr', 'hu', 'id', 'it', 'ja', 'ko', 'lt', 'lv', 'ms', 'mt', 'nb', 'nl', 'pl', 'pt-BR', 'pt', 'ro', 'ru', 'sk', 'sl', 'sv', 'th', 'tr', 'vi', 'zh', 'zh-HK', 'zh-TW'];
     var wca_locale = '<%= I18n.locale %>';
     var stripe = Stripe('<%= EnvVars.STRIPE_PUBLISHABLE_KEY %>', {
       locale: supported_locales.includes(wca_locale) ? wca_locale : 'auto',
       stripeAccount: '<%= @competition.connected_stripe_account_id %>',
     });
-    var elements = stripe.elements();
-    var cardElement = elements.create('card');
-    cardElement.mount('#card-element');
-    var cardholderName = document.getElementById('cardholder-name');
-    var $cardButton = $('#card-button');
-
-    var toggleSaving = function(saving) {
-      $cardButton.attr("disabled", saving);
-      $cardButton.toggleClass("saving", saving);
-    }
+    var $stripeButton = $('#stripe-button');
```

Comment:
Do we want to continue using jQuery for this? If we want to get rid of this I think we should be using document.querySelector instead